.circleci/config.yml

```diff
@@ -41,4 +53,7 @@ workflows:
   WordPress-Utils-Android:
     jobs:
       - Lint
-      - Build
+      - Test
+      - Build:
+          requires:
+            - Test
```

Comment:
As far as I understand, one could argue that if the tests pass, there is no value in also checking that the library builds.

That might be the case now, but when we'll introduce the Bintray publishing, it will be useful to have the build task too (see #40)